bot/src/queue/queues/index.ts

```
@@ -0,0 +1,9 @@
+import { Queue } from 'bullmq'
+import { redisConfig } from '../../config/queue'
+
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class QueueFactory {
+    static createQueue(name: string): Queue {
+        return new Queue(name, { connection: redisConfig })
+    }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)